content/manuals/compose/how-tos/oci-artifact.md

```diff
@@ -0,0 +1,136 @@
+```
+
+To run the Compose application, use the `docker compose` command with the `-f` flag pointing to your OCI artifact:
+```bash
+$ docker compose -f oci://docker.io/username/my-compose-app:latest up
+```
+
+### Warnings/Messages displayed
+
+When you run an application from an OCI artifact, Compose may display warning messages requiring your confirmation to limit risks of running a malicious application:
+* Listing the interpolation variables used along with their values
+* Listing all environment variables used by the application
+* Let you know if your OCI artifact application is using another remote resources (via `include` for example)
+
+```bash 
+$ REGISTRY=myregistry.com docker compose -f oci://docker.io/username/my-compose-app:latest up
+
+Found the following variables in configuration:
+VARIABLE     VALUE                SOURCE        REQUIRED    DEFAULT
+REGISTRY     myregistry.com      command-line   yes         
+TAG          v1.0                environment    no          latest
+DOCKERFILE   Dockerfile          default        no          Dockerfile
+API_KEY      <unset>             none           no          
+
+Do you want to proceed with these variables? [Y/n]:y
+
+Warning: This Compose project includes files from remote sources:
+- oci://registry.example.com/stack:latest
+Remote includes could potentially be malicious. Make sure you trust the source.
+Do you want to continue? [y/N]: 
+```
+
+If you agree to start the application, Compose will display the directory where all the resources from the OCI artifact have been downloaded.
+```bash
+...
+Do you want to continue? [y/N]: y
+
+Your compose stack "oci://registry.example.com/stack:latest" is stored in "~/Library/Caches/docker-compose/964e715660d6f6c3b384e05e7338613795f7dcd3613890cfa57e3540353b9d6d"
+```
+---
+
+## Publishing your Compose application as an OCI artifact
+
+To distribute your Compose application as an OCI artifact, you can **publish** it to an OCI-compliant registry. 
+This allows others to deploy your application directly from the registry.
+
+The publish function supports most of the composition capabilities of Compose, like overrides, extends or include, [with some limitations](#limitations-and-considerations)
+
+### Steps
+
+1. Navigate to Your Compose Application Directory  
+   Ensure you're in the directory containing your `docker-compose.yml` file or that you are specifying your Compose file with the `-f` flag.
+
+2. Log in to Docker Hub
+   Before publishing, make sure you're authenticated with Docker Hub:
+
+   ```bash
+   $ docker login
+   ```
+
+3. Publish the Compose application to Docker Hub  
+   Use the `docker compose publish` command to push your application as an OCI artifact:
+
+   ```bash
+   $ docker compose publish username/my-compose-app:latest
+   ```
+   or passing multiple Compose files
+   ```bash
+   $ docker compose -f docker-compose.yml -f docker-compose.override.yml publish username/my-compose-app:latest
```

Comment:
not a good example, as compose + override are the default files being selected when flag is not set.
Also prefer `compose.yaml` over `docker-compose.yml` according to the spec